gitlab_runner/tests/common.py

```diff
@@ -20,14 +20,15 @@
 
 GITLAB_RUNNER_TAGS = ['gitlab_host:{}'.format(HOST), 'gitlab_port:{}'.format(GITLAB_LOCAL_MASTER_PORT)]
 
+GITLAB_RUNNER_VERSION = os.environ['GITLAB_RUNNER_VERSION']
```

Comment:
Should we update the test setup to also define a `GITLAB_RUNNER_IMAGE` env variable for use in `docker-compose.yml`? Otherwise we'd have the version set in two different places (the Compose setup, and `tox.ini`).